conda_smithy/feedstock_content/.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,17 @@
+Thank you for pull request.
+
+Below are a few things we ask you kindly to self-check before getting a review.
+
+* [ ] Bump the build number (if the version is unchanged)
+* [ ] Reset the build number to `0` (if the version changed)
+* [ ] [Re-render]( https://conda-forge.org/docs/conda_smithy.html#how-to-re-render ) with the latest `conda-smithy`
+* [ ] Updating pinnings based on the [current list]( https://github.com/conda-forge/conda-forge.github.io/blob/master/scripts/pin_the_slow_way.py#L39 )
+* [ ] Ensure the license file is being packaged.
```

Comment:
Do we have a link for what to do if the license file isnt being packaged? (Or how even to check to make sure that it is being packaged?)

Reply:
Looks like [we do]( http://conda-forge.readthedocs.io/en/latest/meta.html#packaging-the-license-manually ). Could add that here if you would like. :)

Reply:
:+1: I think that could be helpful

Reply:
I think this is relevant only for `staged-recipes` right?

Reply:
There are still a number of legacy feedstocks that came in before this started getting enforced more rigorously. Caught one just the other day. ( https://github.com/conda-forge/jupyterlab-feedstock/issues/70 ) So this is still a nice reminder to have in a feedstock PR.

Reply:
If we are enforcing it, maybe that should go into the linter?

Reply:
Generally I agree. However there are a few tricky cases still. Like source codes that have no license file whatsoever. So if we can get the linter to "lighten up" for these or establish some solution for the org, then that seems ok.

Reply:
Okay. I think we should have lints that are suggestions and won't give an error status

Reply:
Definitely. Also should add this now intersects with issues ( https://github.com/conda-forge/conda-smithy/issues/317 ) and ( https://github.com/conda-forge/conda-smithy/issues/591 ).

Reply:
So was giving your original suggestion some more thought. We might benefit from a general license link (which references vendoring the license). There is this [checklist]( https://conda-forge.org/docs/recipe.html#checklist ) in the docs, which [links to this example snippet]( https://github.com/conda-forge/staged-recipes/blob/a504af81c05491bf7b0b018b2fa1efe64767985c/recipes/example/meta.yaml#L52-L55 ). What do you think?

Reply:
Alright, going to save this for a follow-up issue/PR. Feel free to get back to me when you have a chance Eric. :)